recipes/sdl/all/conanfile.py

```diff
@@ -1,6 +1,10 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.apple import is_apple_os
+from conan.tools.files import get, patch, replace_in_file, rm, rmdir
```

Comment:
use other name for applying patchs because it is conflicting with the import name

Reply:
thanks! DIdn't notice that.

Hope I did it right, I'm not a python guy.